spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -1885,10 +1885,8 @@ def foo(a)
   end
 
   context 'when converting a ruby class' do
-    it 'should yield \'PIntegerType\' for Integer, Fixnum, and Bignum' do
-      [Integer,Fixnum,Bignum].each do |c|
-        expect(calculator.type(c).class).to eq(PIntegerType)
-      end
+    it 'should yield \'PIntegerType\' for Integer' do
+      expect(calculator.type(Integer).class).to eq(PIntegerType)
```

Comment:
Isn't that a bit of an overkill? After all, both `Fixnum` and `Bignum` inherits from `Integer`.